cmake_format/format_tests.py

```diff
@@ -2,7 +2,11 @@
 import os
 import difflib
 import unittest
-import StringIO
+
+try:
+  import StringIO
+except ImportError:
+  import io as StringIO
```

Comment:
FYI: Looks like the io module exists in python 2.7 as well..

Reply:
But StringIO doesn't in 3.0+.

> The StringIO and cStringIO modules are gone. Instead, import the io module and use io.StringIO or io.BytesIO for text and data respectively.